### DISCUSSION TITLE
Fails to resume schedules when Macbook lid is closed

### DISCUSSION THREAD

**Discussion by benallfree** [garethgeorge/backrest#189] [Q&A]
I posted here https://forum.restic.net/t/scheduled-backup-failures-after-sleep-on-os-x/7421 but I also had a thought that this might be a Backrest issue. Sometimes after I close my Macbook lid, backrest fails to start any new scheduled backups. Is there anyone here who can help me troubleshoot/test this?

**Comment by garethgeorge** [comment]
Hey, I’m actively investigating a similar issue on Linux. I suspect it’s very likely these are related, take a look at: 

https://github.com/garethgeorge/backrest/issues/172

If you are able to dump a trace of active goroutines next time you see this happen it’d be very helpful to my efforts to debug.
